### PR TITLE
add Asiainfo company support

### PR DESCRIPTION
**Pull request by chenbo90**
## Description

CSFMesh
add Asiainfo company support

### CODE REVIEW

File: data/companies.yml [hunk 1]
```diff
@@ -284,6 +284,12 @@ integrations:
     description: "Space Cloud is a Kubernetes based serverless platform that provides instant, realtime APIs on any database, with event triggers and unified APIs for your custom business logic."
     details:
       - "Space Cloud integrates with Kubernetes and Istio natively to bring to you a highly scalable Serverless Platform. It encrypts all traffic by default and lets you describe communication policies to protect your microservices."
+  - name: "CSFMesh"
+    logo: "/logos/asiainfo.svg"
```
Comment (wilsonwu) [inline data/companies.yml:288]: The image is .png.

**Comment by chenbo90** [comment]
oh , sorry !&nbsp;I will change the image format to PNG. Do I need to submit another PR?




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "istio/istio.io"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年5月21日(星期二) 下午5:59
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [istio/istio.io] add Asiainfo company support (PR #15155)





 
@wilsonwu commented on this pull request.
 
 
In data/companies.yml:
 &gt; @@ -284,6 +284,12 @@ integrations:      description: "Space Cloud is a Kubernetes based serverless platform that provides instant, realtime APIs on any database, with event triggers and unified APIs for your custom business logic."      details:        - "Space Cloud integrates with Kubernetes and Istio natively to bring to you a highly scalable Serverless Platform. It encrypts all traffic by default and lets you describe communication policies to protect your microservices." +  - name: "CSFMesh" +    logo: "/logos/asiainfo.svg"  
The image is .png.

**Comment by wilsonwu** [comment]
> oh , sorry !&nbsp;I will change the image format to PNG. Do I need to submit another PR?
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "istio/istio.io" ***@***.***&gt;; 发送时间:&nbsp;2024年5月21日(星期二) 下午5:59 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [istio/istio.io] add Asiainfo company support (PR #15155) @wilsonwu commented on this pull request. In data/companies.yml: &gt; @@ -284,6 +284,12 @@ integrations: description: "Space Cloud is a Kubernetes based serverless platform that provides instant, realtime APIs on any database, with event triggers and unified APIs for your custom business logic." details: - "Space Cloud integrates with Kubernetes and Istio natively to bring to you a highly scalable Serverless Platform. It encrypts all traffic by default and lets you describe communication policies to protect your microservices." + - name: "CSFMesh" + logo: "/logos/asiainfo.svg" The image is .png. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***&gt;

Just fix it in this PR is OK.

File: data/companies.yml [hunk 2]
```diff
@@ -284,6 +284,12 @@ integrations:
     description: "Space Cloud is a Kubernetes based serverless platform that provides instant, realtime APIs on any database, with event triggers and unified APIs for your custom business logic."
     details:
       - "Space Cloud integrates with Kubernetes and Istio natively to bring to you a highly scalable Serverless Platform. It encrypts all traffic by default and lets you describe communication policies to protect your microservices."
+  - name: "CSFMesh"
+    logo: "/logos/asiainfo.png"
+    url: "https://www.asiainfo.com/zh_cn/product_aisware_aifPaaS.html"
```
Comment (windsonsea) [inline data/companies.yml:289]:
```suggestion
    url: "https://www.asiainfo.com/en_us/product_aisware_aifPaaS.html"
```
Provide an en url please.

File: data/companies.yml [hunk 3]
```diff
@@ -284,6 +284,12 @@ integrations:
...
+    description: "CSFMesh is a service mesh product developed by AsiaInfo Technologies. It features a graphical configuration interface, which enhances operational efficiency."
+    details:
```
Comment (windsonsea) [inline data/companies.yml:291]: Your text in desc and details shall clearly show the connections to Istio.

File: data/companies.yml (hunk 2 above)
Comment (chenbo90) [reply data/companies.yml]: ok , i already change the url

File: data/companies.yml (hunk 3 above)
Comment (chenbo90) [reply data/companies.yml:291]: i already add more details for show connection to istio

File: data/companies.yml [hunk 4]
```diff
@@ -284,6 +284,12 @@ integrations:
     description: "Space Cloud is a Kubernetes based serverless platform that provides instant, realtime APIs on any database, with event triggers and unified APIs for your custom business logic."
     details:
       - "Space Cloud integrates with Kubernetes and Istio natively to bring to you a highly scalable Serverless Platform. It encrypts all traffic by default and lets you describe communication policies to protect your microservices."
+  - name: "CSFMesh"
+    logo: "/logos/asiainfo.png"
```
[the remark this answers is not in the document]
Comment (windsonsea) [reply data/companies.yml:288]: Mostly, yes, .svg is better.

[the remark this answers is not in the document]
Comment (chenbo90) [reply data/companies.yml]: ok , i have already add svg image

**Comment by wilsonwu** [comment]
Suggest you provide a light mode logo, for this white text logo, it looks not good:
image

File: data/companies.yml [hunk 5]
```diff
@@ -284,6 +284,12 @@ integrations:
     description: "Space Cloud is a Kubernetes based serverless platform that provides instant, realtime APIs on any database, with event triggers and unified APIs for your custom business logic."
     details:
       - "Space Cloud integrates with Kubernetes and Istio natively to bring to you a highly scalable Serverless Platform. It encrypts all traffic by default and lets you describe communication policies to protect your microservices."
+  - name: "CSFMesh"
+    logo: "/logos/asiainfo.svg"
+    url: "https://www.asiainfo.com/en_us/product_aisware_aifPaaS.html"
+    description: "CSFMesh is a service mesh product developed by AsiaInfo Technologies. It features a graphical configuration interface, which enhances operational efficiency."
+    details:
+      - "CSFMesh is a dashboard based on istio, which provides functions including cluster management, namespace management and traffic management. Users can easily configure their own traffic management policies and deliver the configuration to the kubernetes cluster. At the same time, we also implemented interface-level service governance and strategies for intercepting traffic on demand."
```
Comment (kfaseela) [inline data/companies.yml:292]:
```suggestion
      - "CSFMesh is a dashboard based on Istio, which provides functions including cluster management, namespace management and traffic management. Users can easily configure their own traffic management policies and apply the configuration to the Kubernetes cluster. CSFMesh also provides interface-level service governance and strategies for intercepting traffic on demand."
```

Comment (chenbo90) [reply data/companies.yml]: thank you very much! my poor english

**Comment by kfaseela** [comment]
@craigbox csfmesh has been added to the integrations section. Hope that is correct?

**Comment by chenbo90** [comment]
> [CLA Missing ID](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/34682364/82706976/15155/#/?version=2) [CLA Not Signed](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/34682364/82706976/15155/#/?version=2)
> 
> * ✅ login: chenbo90  ([547ab8e](https://github.com/istio/istio.io/commit/547ab8efad24956ce55a7e34011ad0e95018bb05), [8629230](https://github.com/istio/istio.io/commit/86292304f694dcae6caf0e2c491c4248e429e9c7))
> * ❌ The email address for the commit ([f280f04](https://github.com/istio/istio.io/commit/f280f048e0775376de980525c6c638372244ba4d), [90759ed](https://github.com/istio/istio.io/commit/90759ed8af9d07c9a385aa8f864d3ad39f31d605), [f4966fc](https://github.com/istio/istio.io/commit/f4966fcd14bcb10e0e58280160c1d6cf147687ba), [4a317cd](https://github.com/istio/istio.io/commit/4a317cd12b30cefb83943c3db3e77a14b2ab63ac), [c02ab96](https://github.com/istio/istio.io/commit/c02ab969ad00e8d41c2a3a16e5c82bc6d5f5ca98), [ef64720](https://github.com/istio/istio.io/commit/ef64720feba8c5b7747992652ea815e113f722d1)) is not linked to the GitHub account, preventing the EasyCLA check. Consult [this Help Article](https://confluence.linuxfoundation.org/pages/viewpage.action?pageId=86641302) and [GitHub Help](https://help.github.com/en/github/committing-changes-to-your-project/why-are-my-commits-linked-to-the-wrong-user) to resolve. (To view the commit's email address, add .patch at the end of this PR page's URL.) For further assistance with EasyCLA, [please submit a support request ticket](https://jira.linuxfoundation.org/servicedesk/customer/portal/4).

----
Two days ago, I used the wrong github account to submit the code, which caused the EasyCLA check to fail. Later I realized the problem. Now I switched to the correct github account and submitted it. Now I see that the EasyCLA check is still reporting errors. What should I do to eliminate this problem?

**Comment by chenbo90** [comment]
Should I close the current PR and resubmit a PR using my correct github account to deal with the EasyCLA inspection problem?

**Comment by craigbox** [comment]
I hope that if you add chenbo3 @ asiainfo.com to your GitHub profile that it would start working

**Pull request closed without merging**